### PR TITLE
Fixes #1462 - Add functional tests for when typing 'g' or 'l' in an issue or issue comment

### LINKED ISSUE #1462: Add functional regression test for preventing shortcut to open label editor when typing comment
Fix was landed in https://github.com/webcompat/webcompat.com/pull/1459, but we should write an Intern test for this.

### PR DESCRIPTION
**Pull request by brizental**
r? @miketaylr 

The tests are failing sometimes, but it's the same thing as I explained in #1470

### CODE REVIEW

File: tests/functional/issues-auth.js [hunk 1]
```diff
@@ -53,7 +53,28 @@ define(
...
+          this,
+          url("/issues/70"),
+          ".wc-Issue-commentSubmit",
+          true
+        )
+          .findByCssSelector(".wc-Issue-labels")
+          .click()
+          .type("l")
+          .end()
+          .setFindTimeout(0)
```
Comment (miketaylr) [inline tests/functional/issues-auth.js]: I think this has the potential to be racy. Can we remove it?  

(It should fall back to the default that `FunctionalHelpers.openPage` uses (`config.wc.pageLoadTimeout`).

File: tests/functional/issues-auth.js [hunk 2]
```diff
@@ -53,7 +53,28 @@ define(
...
             assert.equal(text, "Close Issue", "Button says Close not Reopen");
           });
+      },
+
+      "Pressing 'l' opens the label editor box": function () {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/70"),
+          ".wc-Issue-commentSubmit",
+          true
```
Comment (miketaylr) [inline tests/functional/issues-auth.js]: any reason to request a longer timeout here?

Comment (brizental) [inline tests/functional/issues-auth.js]: oh sorry, i forgot to remove that.
i was just testing to see if the timeout was the problem with the tests that are intermittently not working.
i'll change it back.

File: tests/functional/issues-auth.js (hunk 1 above)
Comment (brizental) [inline tests/functional/issues-auth.js]: these were also tests that i forgot to remove, my bad

**Comment by miketaylr** [comment]
Thanks!

**Pull request merged**